ophyd/__init__.py

```diff
@@ -1,5 +1,3 @@
-# type: ignore
```

Comment:
I'm not sure why this type ignore statement was here, as well as the one at the top of `device.py`.
It was preventing mypy from traversing the submodules properly so I removed it. It didn't seem to break anything in the build, or when I imported it in to our beamline library so maybe its OK?